app/views/foreman_bootdisk/host.erb

```diff
@@ -14,26 +14,29 @@ bootdisk_raise(N_('Subnet (%s) has no gateway defined'), interface.subnet) if in
 bootdisk_raise(N_('Subnet (%s) has no primary DNS server defined'), interface.subnet) if interface.subnet.dns_primary.nil? || interface.subnet.dns_primary.empty?
 %>
 
+echo Foreman Bootdisk: Host image (<%= @host.name %>)
+
 # loop over net* until the host's MAC matches
 <% (0..32).each do |i| -%>
 :net<%= i %>
 isset ${net<%= i -%>/mac} || goto no_nic
 echo net<%= i -%> is a ${net<%= i -%>/chip} with MAC ${net<%= i -%>/mac}
 iseq ${net<%= i -%>/mac} <%= interface.mac -%> || goto net<%= i+1 %>
-ifopen net<%= i %>
```

Comment:
I think this `ifopen` is useless, the device is opened below and I believe double opening without properly closing it via `ifclose` might cause issues with some drivers.